prow.go

```diff
@@ -35,6 +32,8 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/tools/remotecommand"
 	"k8s.io/client-go/transport"
+	"k8s.io/klog"
```

Comment:
Remove this change as unnecessary.

Reply:
My editor automatically sorted imports.  I'll update.